eq-author/src/App/page/Design/answers/BasicAnswer/index.js

```diff
@@ -150,9 +153,14 @@ export const StatelessBasicAnswer = ({
           if (error.errorCode === "PIPING_TITLE_DELETED") {
             message = PIPING_TITLE_DELETED.message;
           }
+          if (error.errorCode === "PIPING_TITLE_MOVED") {
```

Comment:
As there are lots of errorCode values being checked for here, but only one message being generated it would be better to re-write this as case statements. Currently the message can constantly be re-written as it goes.

Also, other constants are prefixed with ERR_ (although not PIPING_TITLE_DELETED). Probably should be consistent with the naming conventions.